app/templates/components/validated-input.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies --}}
```

Comment:
See note above; it's a bit surprising to me that we need to copy this code over. If we can avoid it, that would be strongly preferred.